### PR TITLE
Move RunRC, DeleteRC, and ListPods into the utility module and refactor

### PR DESCRIPTION
**Pull request by rrati**
to be support functions #7234

### CODE REVIEW

File: test/e2e/density.go [hunk 1]
```diff
@@ -359,9 +196,9 @@ var _ = Describe("Density", func() {
 					for i := 0; i < itArg.rcsPerThread; i++ {
 						name := "my-short-lived-pod" + string(util.NewUUID())
 						n := itArg.podsPerMinion * minionCount
-						RunRC(c, name, ns, "gcr.io/google_containers/pause:go", n)
+						expectNoError(RunRC(c, name, ns, "gcr.io/google_containers/pause:go", n))
 						podsLaunched += n
-						glog.Info("Launched %v pods so far...", podsLaunched)
+						Logf("Launched %v pods so far...", podsLaunched)
```
Comment (wojtek-t) [inline test/e2e/density.go]: Just curious: why do you replace it? The "glog" format seems to have more information.

Comment (rrati) [inline test/e2e/density.go]: We want all logging to be done through ginkgo's logging facilities.  That's why Logf was written.

Comment (wojtek-t) [inline test/e2e/density.go]: I see - that's why we are using a different logging in the code itself than in e2e test, right?

Comment (rrati) [inline test/e2e/density.go]: Correct.  Ginkgo allows parallelizing tests and other features and using its logging facility ensures the logs stay with the test and get rolled up in reports generated from ginkgo logs.

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**